server/app/views/components/FieldWithLabel.java

```diff
@@ -428,7 +428,19 @@ private DivTag getNonNumberInputTag() {
     } else {
       throw new RuntimeException("non-number tag expected");
     }
-    return applyAttrsAndGenLabel(inputFieldTag);
+    return applyAttrsClassesAndLabel(inputFieldTag);
+  }
+
+  private DivTag getUSWDSNonNumberInputTag() {
```

Comment:
This is doing almost the same things as getNonNumberInputTag(), can you combine the two? May be pass in a variable or set a class param to identify it as a USWDS component.

Reply:
Done